src/hive/tracer/heartbeat.clj

```diff
@@ -0,0 +1,26 @@
+(ns hive.tracer.heartbeat
+  (:require [chime :refer [chime-ch]]
+            [clj-time.core :as t]
+            [clj-time.periodic :refer [periodic-seq]]
```

Comment:
I really do think we should avoid chime and clj-time to remove dependencies, specially chime